linc_convert/modalities/psoct/multi_slice.py

```diff
@@ -167,23 +172,13 @@ def convert(
     logger.info("Write OME-Zarr multiscale metadata")
     zgroup.write_ome_metadata(axes=["z", "y", "x"], space_unit=to_ome_unit(unit))
 
-    if not zarr_config.nii:
-        logger.info("Conversion complete.")
-        return
+    if nii_config.nii:
+        header = build_nifti_header(
+            zgroup=zgroup,
+            voxel_size_zyx=tuple(vx),
```

Comment:
Just double checking.  It looks like the order of the `vx` elements were previously reversed.  Does it need to be reversed here?

`affine = orientation_to_affine(orientation, *vx[::-1])`